File: code/mathematical_algorithms/src/count_digits/CountDigits.java
/* Part of Cosmos by OpenGenus Foundation */
public class CountDigits {
    public static void main(String args[]) {
        int num = 12345678;
        System.out.println(count_digits(num));
    }
    
    public static int count_digits(int n)
    {
    	if(n == 0) return 1;
    	
        int count = 0;
        while(n != 0){
            count++;
            n /= 10;
        }
        return count;
    }
}

File: code/mathematical_algorithms/src/count_digits/count_digits.py
# Part of Cosmos by OpenGenus Foundation
def count_digits(n):
    if n == 0:
        return 1
    count = 0
    while n != 0:
        count += 1
        n //= 10
    return count


if __name__ == "__main__":
    number = input("Give me a number")
    print(count_digits(number))
